### ISSUE TITLE
redundant halves during deepemhancer step

### ISSUE THREAD

**Issue by hnndlp** [erneyramirez/relion_deepEMhancer_extRec#7]
two python instances are running, one for each half, but each instance launches two deepemhancer jobs (one for each half again).
in total 4 deepemhancer instances are run per iteration.

**Comment by erneyramirez** [comment]
@hnndlp  Yes, and this is also what causes issue #6  (file write error). 
RELION apparently launches two threads at the same time ( something like this) to create the two-half maps with relion_externa_reconstruct. I have tried to stop one of the threads, but then it fails to create the half-maps. This is a pending issue that I have not been able to solve.
Then, when running it twice, deepEMhancer sees that the file is already created. This does not affect the results, but slows down the execution.
